docs/modules/ROOT/pages/how-tos/gke_node_maintenance.adoc

```diff
@@ -0,0 +1,51 @@
+= GKE Node Maintenance
+
+== Node maintenance
+
+GKE allows you to auto-upgrade your Kubernetes control-plane and nodes.
+See https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-upgrades[Cluster upgrades] and https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-upgrades[Auto-upgrading nodes] for more details.
+Auto-Upgrade is enabled for the nodes by default.
+GKE allows you to specify maintenance windows and exclusions to control when an auto-upgrade is executed.
+See https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions[Configuring maintenance windows and exclusions]
+
+=== Change Kubernetes Version
```

Comment:
I somehow don't get what this section wants to tell the reader in regards to the node maintenance. Either it needs clarification or should be deleted.

Reply:
basically it should be a tl;dr for the google docs. I don't want to copy past content.
I can try to clarify it a bit more (without just copy/paste content from the google docs) 
or just put the important links and then let the user read it by himself?

Reply:
I have made some sub header below "Node maintenance" with first for a better structure:
* a tl;dr & links to the important docs from google for the node maintenance 
* and then the actual doing to initiate the version upgrade form the cluster and the nodes

I hope this clairfy's it a bit more?